problems/lonelyNumber.yaml

```diff
@@ -0,0 +1,38 @@
+id: 7
+title: LonelyNumber
+text: |
+  Write a function called findLonely that takes in an array of integer.
+  The given array of integer will always have exactly one unique number, called the lonely number.
+
+  Other numbers than the lonely number will always exist in multiples of two.
+  An example of such an array is [2, 2, 1, 5, 5, 5, 5] where 1 is the lonely number
+
+  findLonely should return the value of the lonely number that it finds.
+  
```

Comment:
Same here, could you insert an example input and output.

findLonely([1, 1, 2, 2, 7, 8, 8]) = 1

Even though the description is good. Some people also prefer to see something like this.

Reply:
I'm thinking to leave this one alone for now as adding an example here seems redundant.

Regarding having an example for users that prefer one, our starter codes have test cases which I think serve that purpose better. Using test cases as examples will also work better in the future as our problems get more complicated to the point where they're impossible to format nicely in the problem description.